service/history/task/transfer_task_executor_base.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/backoff"
 	"github.com/uber/cadence/common/definition"
+	"github.com/uber/cadence/common/elasticsearch/validator"
```

Comment:
elasticseach is a plugin. rest of the codebase shouldn't directly depend on this package. same search attribute constrationts apply to Pinot and other future visibility stores as well. so let's move this validator to `common/visibility` or somewhere generic like that

Reply:
done